src/WingetCreateCore/Common/PackageParser.cs

```diff
@@ -548,5 +644,39 @@ private static string RemoveInvalidCharsFromString(string value)
         {
             return Regex.Replace(value, InvalidCharacters, string.Empty);
         }
+
+        /// <summary>
+        /// DetectedArch object with information about an installer's architectures detected from the url and binary.
+        /// </summary>
+        public class DetectedArch
```

Comment:
>DetectedArch [](http://example.com/codeflow?start=21&length=12)

You can use a record type for this (see PublisherApp.cs)

Reply:
Changed to record type